### ISSUE TITLE
Add record pointing to marathon leader

### ISSUE THREAD

**Issue by discordianfish** [mesosphere/mesos-dns#327]

**Comment by jdef** [comment]
sounds like this might be part of a larger story: frameworks should be able to advertise additional services to mesos-dns (perhaps via plugins?)

**Comment by discordianfish** [comment]
Yes, I guess this deserves a bigger discussion. Will do that with Tomas once he's back. I just wanted to add this issue so I don't forget about it. Was debugging a marathon issue and realized it was too time consuming to figure out the marathon leader.

**Comment by tsenart** [comment]
@discordianfish: Please elaborate on the proposal in the description of the issue.

**Comment by graphex** [comment]
I'm currently on 0.4.0, has this been implemented (0.5.2 being the newest at this time)?
I can get `/v1/hosts/marathon.mesos` which appears to return the marathon leader, but for consistency I would like to get an SRV record for the marathon leader. Based on https://docs.mesosphere.com/usage/service-discovery/mesos-dns/service-naming/ it seems like i should be able to get a record at `/v1/services/_marathon._tcp.marathon.mesos` but am unable to do so in 0.4.0. Unclear if that would return the leader or the set of marathon masters, but API to return both of those from the HTTP API would be very useful.
